File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonQuote.java
package com.zawzaw.savethelibrary.model.gson;

import com.google.gson.annotations.SerializedName;

/**
 * Created by zawzaw on 08/01/18.
 */

public class GsonQuote {

    @SerializedName("id")
    private int id;

    @SerializedName("summery")
    private String summery;

    @SerializedName("content")
    private String content;

    @SerializedName("quoteauthor_name")
    private String quote_author_name;

    @SerializedName("quoteauthor_image")
    private String quote_author_image;

    @SerializedName("reference")
    private String reference;

    public int getId() {
        return id;
    }

    public String getSummery() {
        return summery;
    }

    public String getContent() {
        return content;
    }

    public String getQuote_author_name() {
        return quote_author_name;
    }

    public String getQuote_author_image() {
        return quote_author_image;
    }

    public String getReference() {
        return reference;
    }

}

File: app/src/main/java/com/zawzaw/savethelibrary/model/gson/GsonQuote.kt
package com.zawzaw.savethelibrary.model.gson

import com.google.gson.annotations.SerializedName

/**
 * Created by zawzaw on 08/01/18.
 */

class GsonQuote {

    @SerializedName("id")
    val id: Int = 0

    @SerializedName("summery")
    val summery: String? = null

    @SerializedName("content")
    val content: String? = null

    @SerializedName("quoteauthor_name")
    val quote_author_name: String? = null

    @SerializedName("quoteauthor_image")
    val quote_author_image: String? = null

    @SerializedName("reference")
    val reference: String? = null

}
